<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winAddRemovePar</class>
 <widget class="QWidget" name="winAddRemovePar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lTemp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Øvre temperaturgrænse:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bAbort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afbryd</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbTemp">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="suffix">
    <string>°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHumi">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nedre fugtighedsgrænse:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHumi">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lHead">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Roman Caps</family>
     <pointsize>28</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Indstil parametre</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bSave">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gem</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbBane">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lHul">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hul:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
